pkg/minishift/network/settings_darwin.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"fmt"
+)
+
+func ConfigureNetworking(machineName string, networkSettings NetworkSettings) {
+	fmt.Println(configureIPAddressMessage, configureIPAddressFailure)
```

Comment:
should never be shown, but allows to build an override function.

Reply:
:+1: